--- ffmpeg-0.10.2/configure	2012-01-26 14:15:59.000000000 -0800
+++ ffmpeg-static/build/ffmpeg-0.10.2/configure	2012-04-26 09:09:59.593701590 -0700
@@ -685,6 +685,8 @@
         test "${f}" = "${f#-l}" && flags="$flags $f" || libs="$libs $f"
     done
     check_$type $($filter_cflags $flags) || return
+    extralibs=`echo "$extralibs" | sed s/-lm\s+//`
+    extralibs="$extralibs -lm"
     check_cmd $ld $LDFLAGS $flags -o $TMPE $TMPO $libs $extralibs
 }
 
